### PR TITLE
Add HMR commands

### PR DESCRIPTION
**Pull request by artonge**
Signed-off-by: Louis Chemineau <louis@chmn.me>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,4 +47,4 @@
 		"@nextcloud/stylelint-config": "^1.0.0-beta.0",
 		"@nextcloud/webpack-vue-config": "^4.1.0"
 	}
-}
+}
```
Comment (szaimen) [inline package.json:50]: Intentional change?

Comment (artonge) [reply package.json]: No, just code editor auto-format, fixed

**Review by szaimen — APPROVED** [review]
Honestly I am not sure what this excactly does but 👍

**Review by skjnldsv — APPROVED** [review]

**Review by Pytal — COMMENTED** [review]
Thanks! 🎉

**Pull request merged**